packages/fiori/src/ShellBar.js

```diff
@@ -819,6 +821,34 @@ class ShellBar extends UI5Element {
 		}
 	}
 
+	_updateClonedMenuItems() {
+		if (!this.menuItems.length) {
```

Comment:
If there are no more items, shouldn't we let it delete all clones too (so the next line should always run)?

Reply:
yes, I should just remove the return I think